client/src/component/MilestoneTable.js

```diff
@@ -1,5 +1,9 @@
+import { AppContext } from "../AppContext";
```

Comment:
yes, it exists, we use context, because we may need it in other components as the project is not completed, for example, we need to use userName in another comp to show it at the top of the page as you said.